randomizer/Lists/KasplatLocations.py

```diff
@@ -84,10 +84,10 @@ def setKasplat(self, state=True):
         KasplatLocation(
             name="Japes Kasplat: Underground",
             map_id=Maps.JapesUnderGround,
-            kong_lst=[Kongs.chunky],
+            kong_lst=[Kongs.donkey, Kongs.tiny, Kongs.chunky],
             coords=[427, 20, 456],
             region=Regions.JapesCatacomb,
-            additional_logic=lambda l: (l.pineapple and l.vines) or l.advanced_platforming,
+            additional_logic=lambda l: (l.pineapple and l.vines and l.ischunky) or (l.advanced_platforming and (l.isdonkey or (l.istiny and l.twirl))),
```

Comment:
Revert back to what this was. If it rolls DK or Tiny and advanced platforming is off, it would fail seed gen every time as kong selection isn't based on settings and isn't rerolled upon fill failures